VSCode/armsnippets.json

```diff
@@ -506,6 +506,123 @@
     ],
     "description": "Availability Set"
 },
+"Azure Firewall": {
+    "prefix": "arm-firewall",
+    "body": [
+        "{",
+        "    \"name\": \"${1:myAzureFirewall}\",",
+        "    \"type\": \"Microsoft.Network/azureFirewalls\",",
+        "    \"apiVersion\": \"2019-04-01\",",
+        "    \"location\": \"[resourceGroup().location]\",",
+        "    \"tags\": {},",
+        "    \"properties\": {",
+        "       \"applicationRuleCollections\": [",
+        "           {",
+        "               \"name\": \"${1:appCollectionName}\",",
+        "               \"properties\": {",
+        "                   \"priority\": ${2:100},",
+        "                   \"action\": {",
+        "                       \"type\": \"${3:Allow,Deny}\"",
+        "                   },",
+        "                   \"rules\": [",
+        "                       {",
+        "                           \"name\": \"${4:appRuleName}\",",
+        "                           \"description\": \"${5:ruleDescription}\",",
+        "                           \"sourceAddresses\": [",
+        "                               \"${6:listOfsourceIpAddress}\"",
```

Comment:
Text after the colon is a default value, not a description so a value like "0.0.0.0" (for this particular token) would be better

Reply:
Sure, I wanted to reference that those were integers and not strings, but I will make the change.